pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -1,4 +1,21 @@
-{ fetchFromGitHub, lib, pythonPackages
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, chardet
+, dbus-python
+, docopt
+, lxml
+, netifaces
+, notify2
+, protobuf
+, psutil
+, PyChromecast
+, pygobject2
+, pyroute2
+, requests
+, setproctitle
+, setuptools
+, zeroconf
 , mp3Support ? true, lame ? null
 , opusSupport ? true, opusTools ? null
 , faacSupport ? false, faac ? null
```

Comment:
```suggestion
, mp3Support ? true, lame
, opusSupport ? true, opusTools
, faacSupport ? false, faac
```

Reply:
Why these 3 and not the rest?

Reply:
Because github did not allow me to do that :P.

Reply:
Addressed in 03edc17